src/argilla/client/feedback/schemas/questions.py

```diff
@@ -334,21 +347,35 @@ class SpanQuestion(QuestionSchema):
 
     Examples:
         >>> from argilla.client.feedback.schemas.questions import SpanQuestion
-        >>> SpanQuestion(name="span_question", title="Span Question", labels=["person", "org"])
+        >>> SpanQuestion(name="span_question", field="prompt", title="Span Question", labels=["person", "org"])
     """
 
     type: Literal[QuestionTypes.span] = Field(QuestionTypes.span, allow_mutation=False, const=True)
 
-    labels: conlist(Union[str, SpanLabelOption], min_items=1, unique_items=True)
+    field: str = Field(..., description="The field in the input that the user will be asked to annotate.")
+    labels: Union[Dict[str, str], conlist(Union[str, SpanLabelOption], min_items=1, unique_items=True)]
+
+    @validator("labels", pre=True)
+    def parse_labels_dict(cls, labels) -> List[SpanLabelOption]:
+        if isinstance(labels, dict):
+            return [SpanLabelOption(value=label, text=text) for label, text in labels.items()]
+        return labels
 
     @validator("labels", always=True)
     def normalize_labels(cls, v: List[Union[str, SpanLabelOption]]) -> List[SpanLabelOption]:
         return [SpanLabelOption(value=label, text=label) if isinstance(label, str) else label for label in v]
 
+    @validator("labels")
```

Comment:
wasn't there a max for the labels too which we defined on the server side? perhaps we can use it here as well?

Reply:
Yes, this is not hard to define.

Reply:
Anyway, this is not the current behaviour that exists for single and multi-label settings. In both cases, there is a min validation but not a max one.

Also, if we plan to support a configurable value for this, adding a hard validation here may introduce workflow problems. So, I will let as is.